docs/support/troubleshooting.md

```diff
@@ -1,11 +1,20 @@
 # Troubleshooting
 
-## How to resolve errors of the setup script?
+- [Setup script](#setup-script)
+  - [CUDA-related errors](#cuda-related-errors)
+- [Build errors](#build-errors)
+  - [Insufficient memory](#insufficent-memory)
+  - [Errors when using the latest version of Autoware](#errors-when-using-the-latest-version-of-autoware)
+  - [Errors when using a fixed version of Autoware](#errors-when-using-a-fixed-version-of-autoware)
+- [Docker/rocker issues](#dockerrocker-issues)
+- [Runtime problems](#runtime-problems)
+  - [Map does not display when running the Planning Simulator](#map-does-not-display-when-running-the-planning-simulator)
```

Comment:
I think it's not good to create such a table of contents for troubleshooting because the number of sections will gradually increase.

Also, we have the TOC on the right of the page.
![image](https://user-images.githubusercontent.com/31987104/168236036-ff9567fa-bece-43f9-a681-489009c3093f.png)

Reply:
Good point. I will remove it.